### PR TITLE
Consolidated tsinfo chromsizes

### PR DESCRIPTION
**Pull request by pkerpedjiev**
## Description

- Return chromsizes as an array instead of as two separate tab-separated fields (`chrom_sizes` and `chrom_names`). This harmonizes the return type with the return type for cooler tileset_info.

Fixes #\_\_\_

## Checklist

-   [x] Unit tests added or updated
-   [x] Updated CHANGELOG.md
-   [x] Run `black .`

### CODE REVIEW

File: test/tiles/npvector_test.py [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import numpy as np
+
 import clodius.tiles.npvector as hgnv
 
 
 def test_npvector():
-    array = np.array(range(100))
+    array = np.array([float(f) for f in range(100)])
```
Comment (pkerpedjiev) [inline test/tiles/npvector_test.py:7]: This is fixing an unrelated issue in the tests.

File: test/mrmatrix_test.py [hunk 2]
```diff
@@ -98,28 +96,24 @@ def test_bins(self):
...
-                    'values': np.array([[3,4],[5,6]])
-                },
-                '11': {
-                    'values': np.array([[5,6],[7,8]])
+        tileset = AttrDict(
+            {
+                "resolutions": {
+                    # TODO: It's not actually enforced that zoom levels be sequential integers?
+                    # TODO: Should we check that the sizes are reasonable during initialization?
+                    "1": {"values": np.array([[1.0, 2.0], [3.0, 4.0]])},
```
Comment (pkerpedjiev) [inline test/mrmatrix_test.py:111]: This is fixing an unrelated issue in the tests.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+v0.16.0
```
Comment (flekschas) [inline CHANGELOG.md:1]: This is a breaking change. Should this be v1? Or maybe we can at least mention it in the changelog that this is going to cause trouble for anyone who relies on `chrom_names` and `chrom_sizes`.

File: clodius/tiles/beddb.py [hunk 4]
```diff
@@ -23,15 +23,16 @@ def tileset_info(db_file):
...
         "tile_size": row[5],
         "max_zoom": row[6],
         "max_width": row[7],
         "min_pos": [1],
         "max_pos": [row[1]],
         "header": header,
         "version": version,
+        "chromsizes": list(
+            zip(row[3].split("\t"), [int(cs) for cs in row[4].split("\t")])
+        ),
```
Comment (flekschas) [inline clodius/tiles/beddb.py:35]:
```suggestion
        ),
        "info_version": "2"
```

I would probably add a versioning to this

**Review by flekschas — APPROVED** [review]
Looks good. Just left two comments

File: CHANGELOG.md (hunk 3 above)
Comment (pkerpedjiev) [reply CHANGELOG.md:1]: I'll mention it in the changelog. There should be a v1 but I'm not sure that this is the moment to declare it.

**Pull request merged**